subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -179,3 +179,33 @@ dependencies {
         api(libs.xmlunit)               { version { strictly("1.6") }}
     }
 }
+
+/** `com.github.mwiede:jsch` is a drop-in replacement for `com.jcraft:jsch` */
+@CacheableRule
+object JschReplacementVersionRule : ComponentMetadataRule {
```

Comment:
I had to copy this rule twice in

* `subprojects/distributions-dependencies/build.gradle.kts`
* `build-logic/build-platform/build.gradle.kts`

Is there a way to share it between all of the relevant java-platform plugins?